Extensions/xkit_preferences.js

```diff
@@ -1680,8 +1680,8 @@ XKit.extensions.xkit_preferences = new Object({
 
 			var has_css = m_extension.css !== "";
 			var has_icon = m_extension.icon !== "";
-			var is_beta = m_extension.beta === true;
-			var is_frame = m_extension.frame === true;
+			var is_beta = m_extension.beta;
+			var is_frame = m_extension.frame;
```

Comment:
I'm not sure if the original intent is preserved here. Note to self: investigate `m_extension`